Enhance comments for RSA key types.

Index: ssh/sshd_config
===================================================================
--- ssh.orig/sshd_config
+++ ssh/sshd_config
@@ -46,7 +46,9 @@
 #MaxSessions 10
 
 # anchor_1i2w6k7g8dtsxb62ev3eliju4_4
+# Pure RSAAuthentication is used for V1 only
 #RSAAuthentication yes
+# PubkeyAuthentication is used for V2 only
 #PubkeyAuthentication yes
 
 # The default is to check both .ssh/authorized_keys and .ssh/authorized_keys2
