Index: ssh/sshd_config
===================================================================
--- ssh.orig/sshd_config
+++ ssh/sshd_config
@@ -158,4 +158,5 @@ Ciphers blowfish-cbc
 # macs: anchor_1i2w6k7g8dtsxb62ev3eliju4_18
 MACs hmac-ripemd160,hmac-sha1
 # kex: anchor_1i2w6k7g8dtsxb62ev3eliju4_19
+KexAlgorithms diffie-hellman-group14-sha1
 # EOF: anchor_1i2w6k7g8dtsxb62ev3eliju4_20
Index: ssh/ssh_config
===================================================================
--- ssh.orig/ssh_config
+++ ssh/ssh_config
@@ -44,6 +44,7 @@ Ciphers blowfish-cbc,3des-cbc
 # macs: anchor_1i2w6k7g8dtsxb62ev3eliju4_4
 MACs hmac-ripemd160,hmac-sha1
 # kex: anchor_1i2w6k7g8dtsxb62ev3eliju4_5
+KexAlgorithms diffie-hellman-group14-sha1
 #   EscapeChar ~
 #   Tunnel no
 #   TunnelDevice any:any
