Select MAC algorithms allowed for this SSH server.

Index: ssh/sshd_config
===================================================================
--- ssh.orig/sshd_config
+++ ssh/sshd_config
@@ -160,6 +160,7 @@ AllowGroups sshlogin
 # ciphers: anchor_1i2w6k7g8dtsxb62ev3eliju4_17
 Ciphers blowfish-cbc
 # macs: anchor_1i2w6k7g8dtsxb62ev3eliju4_18
+MACs hmac-ripemd160,hmac-sha1
 # kex: anchor_1i2w6k7g8dtsxb62ev3eliju4_19
 # kick-stale: anchor_1i2w6k7g8dtsxb62ev3eliju4_21
 ClientAliveInterval 1800
Index: ssh/ssh_config
===================================================================
--- ssh.orig/ssh_config
+++ ssh/ssh_config
@@ -42,7 +42,7 @@ Cipher 3des
 Ciphers blowfish-cbc,3des-cbc
 # hkalgos: anchor_1i2w6k7g8dtsxb62ev3eliju4_3
 # macs: anchor_1i2w6k7g8dtsxb62ev3eliju4_4
-#   MACs hmac-md5,hmac-sha1,umac-64@openssh.com,hmac-ripemd160
+MACs hmac-ripemd160,hmac-sha1
 # kex: anchor_1i2w6k7g8dtsxb62ev3eliju4_5
 #   EscapeChar ~
 #   Tunnel no
