Customized the "nocheck" parameter.

Index: layman/layman.cfg
===================================================================
--- layman.orig/layman.cfg
+++ layman/layman.cfg
@@ -45,7 +45,7 @@ overlays  : http://www.gentoo.org/proj/e
 # warnings if an overlay definition is missing either 
 # description or contact information.
 #
-nocheck  : yes
+nocheck  : no
 
 #-----------------------------------------------------------
 # Umask settings
