Adds locale/location defaults for country "Austria".

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -457,13 +457,13 @@ DOC_SYMLINKS_DIR="/usr/share/doc/html"
 # which the installers did not disable translation at _configure_ time. Then,
 # users only have to set the 'LANG' environment variable to the appropriate
 # 'LL_CC' combination prior to using the programs in the package.
-#LANG=""
+LANG="de_AT.utf8"
 
 # For GNU gettext based packages, this selects which of the available
 # translation files of a package should be installed. Some packages like OOo
 # also use this to select the language to be hardcoded. See
 # >>>info "(gettext)po/LINGUAS"<<< for details.
-#LINGUAS=""
+LINGUAS="de en@boldquot en@quot"
 
 # The command used to compress documentation during the install phase.
 PORTAGE_COMPRESS="lzma"
@@ -523,7 +523,7 @@ PORTAGE_WORKDIR_MODE="0705"
 # weak algorithms or key lengths might be chosen by the installer.
 SSL_BITS="2048"
 #SSL_COMMONNAME="First Last"
-#SSL_COUNTRY="AT"
+SSL_COUNTRY="AT"
 SSL_DAYS="730"
 SSL_EMAIL="root@localhost"
 #SSL_LOCALITY="City"
