Adds locale/location defaults for country "Austria".

Index: merging/make.conf
===================================================================
--- merging.orig/make.conf
+++ merging/make.conf
@@ -498,10 +498,10 @@ DOC_SYMLINKS_DIR="/usr/share/doc/html"
 # Then, users only have to set the 'LANG' environment variable to the
 # appropriate 'LL_CC' combination prior to using the programs in the
 # package.
-#LANG=""
+LANG="de_AT.utf8"
 
 # Packages like OOo use this to select the language to be hardcoded.
-#LINGUAS=""
+LINGUAS="de"
 
 # If this is set, it should contain the path to a GnuPG public key ring.
 # The keys in that keyring will then be used for cryptographic
@@ -522,7 +522,7 @@ PORTAGE_WORKDIR_MODE="0705"
 # weak algorithms or key lengths might be chosen by the installer.
 SSL_BITS="2048"
 #SSL_COMMONNAME="First Last"
-#SSL_COUNTRY="AT"
+SSL_COUNTRY="AT"
 SSL_DAYS="730"
 SSL_EMAIL="root@localhost"
 #SSL_LOCALITY="City"
