Disable automatic mounting of /boot when running GRUB.

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -444,6 +444,9 @@ DOC_SYMLINKS_DIR="/usr/share/doc/html"
 # each time an ebuild accessing one is re-emerged.
 #EBZR_OFFLINE="1"
 
+# Used by GRUB ebuild in order to avoid automounting /boot.
+DONT_MOUNT_BOOT="1"
+
 # User and e-mail for Gentoo's changelog authoring tool.
 # Only required if you are actually using that tool.
 ECHANGELOG_USER="Guenther Brunthaler <gb_about_gnu@gmx.net>"
