Define basic settings for "xtreme.xworld.mine.nu".

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -35,7 +35,7 @@ USE="
 # Change this line as appropriate (i686, i586, i486 or i386).
 # All modern systems (even Athlons) should use "i686-pc-linux-gnu".
 # All K6's are i586.
-#CHOST="i686-pc-linux-gnu"
+CHOST="i686-pc-linux-gnu"
 
 # Host and optimization settings 
 # ==============================
@@ -106,6 +106,7 @@ USE="
 #CFLAGS="-mcpu=athlon-xp -O2 -pipe"
 #CFLAGS="-march=pentium3 -O2 -pipe"
 # Avoid -O0 and -s in CFLAGS.
+CFLAGS="-march=nocona -O3 -DNDEBUG -pipe -fomit-frame-pointer -fno-stack-check"
 
 # If you set a CFLAGS above, then this line will set your default C++ flags to
 # the same settings.
@@ -264,6 +265,7 @@ source /var/lib/layman/make.conf
 #     http://www.gentoo.org/doc/en/rsync.xml
 #
 #SYNC="rsync://rsync.gentoo.org/gentoo-portage"
+SYNC="rsync://rsync.de.gentoo.org/gentoo-portage"
 #
 # PORTAGE_RSYNC_RETRIES sets the number of times portage will attempt to retrieve
 #     a current portage tree before it exits with an error. This allows
@@ -274,7 +276,7 @@ source /var/lib/layman/make.conf
 #     command used by `emerge --sync`. This will not change the default options
 #     which are set by PORTAGE_RSYNC_OPTS (don't change those unless you know 
 #     exactly what you're doing).
-#PORTAGE_RSYNC_EXTRA_OPTS=""
+PORTAGE_RSYNC_EXTRA_OPTS="--bwlimit=100"
 #
 # Advanced Features
 # =================
@@ -295,6 +297,7 @@ EMERGE_DEFAULT_OPTS="--nospinner --with-
 #     the number of parallel makes (-j) to perform. The suggested number
 #     for parallel makes is CPUs+1.
 #MAKEOPTS="-j2"
+MAKEOPTS="-j1"
 #
 # PORTAGE_NICENESS provides a default increment to emerge's niceness level.
 #     Note: This is an increment. Running emerge in a niced environment will
@@ -450,7 +453,7 @@ ECHANGELOG_USER="Guenther Brunthaler <gb
 # If you are using X11, setting this variable will restrict
 # the set of input device drivers to be compiled in.
 # Otherwise, all supported input drivers will be included.
-#INPUT_DEVICES="evdev keyboard mouse"
+INPUT_DEVICES="evdev keyboard mouse vmmouse"
 
 # The directory containing kernel the target kernel sources.
 # Defaults to symlink "/usr/src/linux" (which is managed by "eselect kernel").
@@ -559,8 +562,10 @@ SSL_UNIT="Preliminary usage"
 # the video adapter cards to be compiled in.
 # Otherwise, all supported video adapter cards will be included.
 #VIDEO_CARDS="fbdev vesa vga"
+VIDEO_CARDS="vmware"
 
 # This entry should be the last in /etc/make.conf!
 #
 # Run "mirrorselect -s10 -b10" in order to update the next line.
 #GENTOO_MIRRORS=""
+GENTOO_MIRRORS="ftp://mirror.bih.net.ba/gentoo/ ftp://ftp.twaren.net/Linux/Gentoo/ http://ftp.twaren.net/Linux/Gentoo/ ftp://de-mirror.org/distro/gentoo/ http://ftp.spline.inf.fu-berlin.de/mirrors/gentoo/ ftp://mirror.netcologne.de/gentoo/ ftp://mirror.icis.pcz.pl/gentoo/ http://mirror.averse.net/pub/gentoo/ http://mirror.its.uidaho.edu/pub/gentoo/ http://mirror.switch.ch/ftp/mirror/gentoo/"
