Define basic settings for "vmware.sciencesoft.at".

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -31,7 +31,7 @@ USE="
 # Host Setting
 # ============
 #
-#CHOST="i686-pc-linux-gnu"
+CHOST="x86_64-pc-linux-gnu"
 
 # Host and optimization settings 
 # ==============================
@@ -72,6 +72,7 @@ USE="
 #CFLAGS="-mcpu=athlon-xp -O2 -pipe"
 #CFLAGS="-march=pentium3 -O2 -pipe"
 # Avoid -O0 and -s in CFLAGS.
+CFLAGS="-march=nocona -O3 -pipe -DNDEBUG -fno-stack-check"
 
 # If you set a CFLAGS above, then this line will set your default C++ flags to
 # the same settings.
@@ -229,7 +230,7 @@ source /var/lib/layman/make.conf
 #     Instructions for setting up a local rsync server are available here:
 #     http://www.gentoo.org/doc/en/rsync.xml
 #
-#SYNC="rsync://rsync.gentoo.org/gentoo-portage"
+SYNC="rsync://rsync.de.gentoo.org/gentoo-portage"
 #
 # PORTAGE_RSYNC_RETRIES sets the number of times portage will attempt to retrieve
 #     a current portage tree before it exits with an error. This allows
@@ -260,7 +261,7 @@ EMERGE_DEFAULT_OPTS="--nospinner --with-
 #     program is compiled. Presently the only use is for specifying
 #     the number of parallel makes (-j) to perform. The suggested number
 #     for parallel makes is CPUs+1.
-#MAKEOPTS="-j2"
+MAKEOPTS="-j5"
 #
 # PORTAGE_NICENESS provides a default increment to emerge's niceness level.
 #     Note: This is an increment. Running emerge in a niced environment will
@@ -527,3 +528,4 @@ SSL_UNIT="Preliminary usage"
 #
 # Run "mirrorselect -s10 -b10" in order to update the next line.
 #GENTOO_MIRRORS=""
+GENTOO_MIRRORS="/usr/portage/local/distfiles/local /usr/portage/local/distfiles /usr/portage/local/distfiles/precious /usr/portage/local/distfiles/mnt ftp://gentoo.in.th/ http://mirror.jamit.de/gentoo/ ftp://ftp.snt.utwente.nl/pub/os/linux/gentoo http://ftp-stud.fht-esslingen.de/pub/Mirrors/gentoo/ http://mirror.its.uidaho.edu/pub/gentoo/ ftp://mirror.its.uidaho.edu/gentoo/ ftp://mirror.pacific.net.au/linux/Gentoo ftp://mirror.iawnet.sandia.gov/pub/gentoo/ ftp://mirror.mdfnet.se/gentoo http://mirrors.xservers.ro/gentoo/"
