Define basic settings for "xquad.xworld.mine.nu".

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -35,7 +35,7 @@ USE="
 # Change this line as appropriate (i686, i586, i486 or i386).
 # All modern systems (even Athlons) should use "i686-pc-linux-gnu".
 # All K6's are i586.
-#CHOST="i686-pc-linux-gnu"
+CHOST="x86_64-pc-linux-gnu"
 
 # Host and optimization settings 
 # ==============================
@@ -106,6 +106,8 @@ USE="
 #CFLAGS="-mcpu=athlon-xp -O2 -pipe"
 #CFLAGS="-march=pentium3 -O2 -pipe"
 # Avoid -O0 and -s in CFLAGS.
+##CFLAGS="-march=k8 -O3 -DNDEBUG -pipe -fno-stack-check"
+CFLAGS="-march=amdfam10 -O3 -pipe -fno-stack-check"
 
 # If you set a CFLAGS above, then this line will set your default C++ flags to
 # the same settings.
@@ -270,6 +272,7 @@ FETCHCOMMAND="/usr/bin/getdelta.sh \"\${
 #     http://www.gentoo.org/doc/en/rsync.xml
 #
 #SYNC="rsync://rsync.gentoo.org/gentoo-portage"
+SYNC="rsync://rsync.de.gentoo.org/gentoo-portage"
 #
 # PORTAGE_RSYNC_RETRIES sets the number of times portage will attempt to retrieve
 #     a current portage tree before it exits with an error. This allows
@@ -301,6 +304,7 @@ EMERGE_DEFAULT_OPTS="--nospinner --jobs=
 #     the number of parallel makes (-j) to perform. The suggested number
 #     for parallel makes is CPUs+1.
 #MAKEOPTS="-j2"
+MAKEOPTS="-j5 -l5"
 #
 # PORTAGE_NICENESS provides a default increment to emerge's niceness level.
 #     Note: This is an increment. Running emerge in a niced environment will
@@ -437,7 +441,7 @@ PORTAGE_ELOG_SYSTEM="save"
 # If you are using ALSA, this variable must be set to the names
 # of the sound cards to be supported. See the "ALSA Sound Matrix"
 # decription in the Internet for a list of possible names.
-#ALSA_CARDS=""
+ALSA_CARDS="hda-intel"
 
 # Add files and directories here which should be put under
 # Gentoo's config protection. Such files will never be
@@ -488,7 +492,7 @@ EVCS_OFFLINE="1"
 # If you are using X11, setting this variable will restrict
 # the set of input device drivers to be compiled in.
 # Otherwise, all supported input drivers will be included.
-#INPUT_DEVICES="evdev keyboard mouse"
+INPUT_DEVICES="evdev joystick keyboard mouse void"
 
 # The directory containing kernel the target kernel sources.
 # Defaults to symlink "/usr/src/linux" (which is managed by "eselect kernel").
@@ -612,7 +616,7 @@ SSL_UNIT="Preliminary usage"
 # If you are using X11, setting this variable will restrict
 # the video adapter cards to be compiled in.
 # Otherwise, all supported video adapter cards will be included.
-#VIDEO_CARDS="fbdev vesa vga"
+VIDEO_CARDS="dummy ati fglrx v4l vesa vga"
 
 # This entry should be the last in /etc/make.conf!
 #
