Define basic settings for "xtreme.xworld.mine.nu".

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -33,7 +33,7 @@ USE="
 # Change this line as appropriate (i686, i586, i486 or i386).
 # All modern systems (even Athlons) should use "i686-pc-linux-gnu".
 # All K6's are i586.
-#CHOST="i686-pc-linux-gnu"
+CHOST="i686-pc-linux-gnu"
 
 # Host and optimization settings 
 # ==============================
@@ -104,6 +104,7 @@ USE="
 #CFLAGS="-mcpu=athlon-xp -O2 -pipe"
 #CFLAGS="-march=pentium3 -O2 -pipe"
 # Avoid -O0 and -s in CFLAGS.
+CFLAGS="-march=pentium4 -O2 -DNDEBUG -pipe -fno-stack-check"
 
 # If you set a CFLAGS above, then this line will set your default C++ flags to
 # the same settings.
@@ -260,6 +261,7 @@ FETCHCOMMAND="/usr/bin/getdelta.sh \${UR
 #     http://www.gentoo.org/doc/en/rsync.xml
 #
 #SYNC="rsync://rsync.gentoo.org/gentoo-portage"
+SYNC="rsync://rsync.de.gentoo.org/gentoo-portage"
 #
 # PORTAGE_RSYNC_RETRIES sets the number of times portage will attempt to retrieve
 #     a current portage tree before it exits with an error. This allows
@@ -291,6 +293,7 @@ EMERGE_DEFAULT_OPTS="--nospinner --with-
 #     the number of parallel makes (-j) to perform. The suggested number
 #     for parallel makes is CPUs+1.
 #MAKEOPTS="-j2"
+MAKEOPTS="-j2"
 #
 # PORTAGE_NICENESS provides a default increment to emerge's niceness level.
 #     Note: This is an increment. Running emerge in a niced environment will
@@ -475,7 +478,7 @@ PORTAGE_ELOG_SYSTEM="save"
 # If you are using ALSA, this variable must be set to the names
 # of the sound cards to be supported. See the "ALSA Sound Matrix"
 # decription in the Internet for a list of possible names.
-#ALSA_CARDS=""
+ALSA_CARDS="intel8x0"
 
 # Add files and directories here which should be put under
 # Gentoo's config protection. Such files will never be
@@ -498,7 +501,7 @@ ECHANGELOG_USER="Guenther Brunthaler <gb
 # If you are using X11, setting this variable will restrict
 # the set of input device drivers to be compiled in.
 # Otherwise, all supported input drivers will be included.
-#INPUT_DEVICES="evdev keyboard mouse"
+INPUT_DEVICES="evdev joystick keyboard mouse synaptics"
 
 # The directory containing kernel the target kernel sources.
 # Defaults to symlink "/usr/src/linux" (which is managed by "eselect kernel").
@@ -612,6 +615,7 @@ SSL_UNIT="Preliminary usage"
 # the video adapter cards to be compiled in.
 # Otherwise, all supported video adapter cards will be included.
 #VIDEO_CARDS="fbdev vesa vga"
+VIDEO_CARDS="fbdev radeon vesa vga"
 
 # This entry should be the last in /etc/make.conf!
 #
