Adds support for the "layman" Portage overlay manager.

Index: merging/make.conf
===================================================================
--- merging.orig/make.conf
+++ merging/make.conf
@@ -182,9 +182,9 @@ PORT_LOGDIR="/var/log/ebuilds"
 # PORTDIR_OVERLAY is a directory where local ebuilds may be stored without
 #     concern that they will be deleted by rsync updates. Default is not
 #     defined.
-#PORTDIR_OVERLAY=/usr/local/portage
+PORTDIR_OVERLAY="/usr/local/portage"
 # Add more overlays (from "layman").
-#source /usr/portage/local/layman/make.conf
+source /usr/portage/local/layman/make.conf
 
 # Fetching files 
 # ==============
