Adds support for the "layman" Portage overlay manager.

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -151,9 +151,9 @@ PORT_LOGDIR="/var/log/ebuilds"
 # PORTDIR_OVERLAY is a directory where local ebuilds may be stored without
 #     concern that they will be deleted by rsync updates. Default is not
 #     defined.
-#PORTDIR_OVERLAY=/usr/local/portage
+PORTDIR_OVERLAY="/usr/local/portage/overlay"
 # Add more overlays (from "layman").
-#source /usr/local/portage/layman/make.conf
+source /var/lib/layman/make.conf
 
 # Fetching files 
 # ==============
