Optimize settings for system administration via SSH.

Index: merging/make.conf
===================================================================
--- merging.orig/make.conf
+++ merging/make.conf
@@ -279,7 +279,7 @@ PORT_LOGDIR="/var/log/ebuilds"
 #     specified on every run. Useful options include --ask, --verbose,
 #     --usepkg and many others. Options that are not useful, such as --help,
 #     are not filtered.
-#EMERGE_DEFAULT_OPTS=""
+EMERGE_DEFAULT_OPTS="--nospinner"
 #
 # INSTALL_MASK allows certain files to not be installed into your file system.
 #     This is useful when you wish to filter out a certain set of files from
