Changes settings regarding the administrative contact to refer
to the person "Guenther Brunthaler".

Index: portage/make.conf
===================================================================
--- portage.orig/make.conf
+++ portage/make.conf
@@ -456,7 +456,7 @@ EBZR_OFFLINE="1"
 
 # User and e-mail for Gentoo's changelog authoring tool.
 # Only required if you are actually using that tool.
-#ECHANGELOG_USER=""
+ECHANGELOG_USER="Guenther Brunthaler <gb_about_gnu@gmx.net>"
 
 # Disable automatic updating of checked-out CVS check-outs
 # each time an ebuild accessing one is re-emerged.
@@ -597,12 +597,12 @@ PORTAGE_WORKDIR_MODE="0705"
 # in the absence of appropriate default settings here, cryptographically
 # weak algorithms or key lengths might be chosen by the installer.
 SSL_BITS="2048"
-#SSL_COMMONNAME="First Last"
+SSL_COMMONNAME="Guenther Brunthaler"
 SSL_COUNTRY="AT"
 SSL_DAYS="730"
 SSL_EMAIL="root@localhost"
 SSL_LOCALITY="Vienna"
-#SSL_ORGANIZATION="My Company"
+SSL_ORGANIZATION="Guenther Brunthaler EDV-Dienstleistungen"
 SSL_STATE="Vienna"
 SSL_UNIT="Preliminary usage"
 
