Define basic settings for "xtreme.xworld.mine.nu".

Index: portage/make.conf
===================================================================
--- portage.orig/make.conf
+++ portage/make.conf
@@ -35,7 +35,7 @@ USE="
 # Change this line as appropriate (i686, i586, i486 or i386).
 # All modern systems (even Athlons) should use "i686-pc-linux-gnu".
 # All K6's are i586.
-#CHOST="i686-pc-linux-gnu"
+CHOST="i686-pc-linux-gnu"
 
 # Host and optimization settings 
 # ==============================
@@ -106,6 +106,7 @@ USE="
 #CFLAGS="-mcpu=athlon-xp -O2 -pipe"
 #CFLAGS="-march=pentium3 -O2 -pipe"
 # Avoid -O0 and -s in CFLAGS.
+CFLAGS="-march=nocona -O3 -pipe -fomit-frame-pointer -fno-stack-check"
 
 # If you set a CFLAGS above, then this line will set your default C++ flags to
 # the same settings.
@@ -270,6 +271,7 @@ source /var/lib/layman/make.conf
 #     http://www.gentoo.org/doc/en/rsync.xml
 #
 #SYNC="rsync://rsync.gentoo.org/gentoo-portage"
+SYNC="rsync://rsync.de.gentoo.org/gentoo-portage"
 #
 # PORTAGE_RSYNC_RETRIES sets the number of times portage will attempt to retrieve
 #     a current portage tree before it exits with an error. This allows
@@ -280,7 +282,7 @@ source /var/lib/layman/make.conf
 #     command used by `emerge --sync`. This will not change the default options
 #     which are set by PORTAGE_RSYNC_OPTS (don't change those unless you know 
 #     exactly what you're doing).
-#PORTAGE_RSYNC_EXTRA_OPTS=""
+PORTAGE_RSYNC_EXTRA_OPTS="--bwlimit=100"
 #
 # Advanced Features
 # =================
@@ -289,7 +291,7 @@ source /var/lib/layman/make.conf
 #     specified on every run. Useful options include --ask, --verbose,
 #     --usepkg and many others. Options that are not useful, such as --help,
 #     are not filtered.
-EMERGE_DEFAULT_OPTS="--nospinner --jobs=8 --with-bdeps=y"
+EMERGE_DEFAULT_OPTS="--nospinner --quiet --with-bdeps=y"
 #
 # INSTALL_MASK allows certain files to not be installed into your file system.
 #     This is useful when you wish to filter out a certain set of files from
@@ -301,6 +303,7 @@ EMERGE_DEFAULT_OPTS="--nospinner --jobs=
 #     the number of parallel makes (-j) to perform. The suggested number
 #     for parallel makes is CPUs+1.
 #MAKEOPTS="-j2"
+MAKEOPTS="-j1"
 #
 # PORTAGE_NICENESS provides a default increment to emerge's niceness level.
 #     Note: This is an increment. Running emerge in a niced environment will
@@ -488,7 +491,7 @@ EVCS_OFFLINE="1"
 # If you are using X11, setting this variable will restrict
 # the set of input device drivers to be compiled in.
 # Otherwise, all supported input drivers will be included.
-#INPUT_DEVICES="evdev keyboard mouse"
+INPUT_DEVICES="evdev keyboard mouse vmmouse"
 
 # The directory containing kernel the target kernel sources.
 # Defaults to symlink "/usr/src/linux" (which is managed by "eselect kernel").
@@ -613,6 +616,7 @@ SSL_UNIT="Preliminary usage"
 # the video adapter cards to be compiled in.
 # Otherwise, all supported video adapter cards will be included.
 #VIDEO_CARDS="fbdev vesa vga"
+VIDEO_CARDS="vmware"
 
 # This entry should be the last in /etc/portage/make.conf!
 #
@@ -622,3 +626,4 @@ SSL_UNIT="Preliminary usage"
 # $ sed -e '/^GENTOO_MIRRORS=/,$ d' "$m" > "$t2"
 # $ cat "$t2" "$t" > "$m" && rm "$t2" "$t" && unset m t t2
 #GENTOO_MIRRORS=""
+GENTOO_MIRRORS="http://gd.tuwien.ac.at/opsys/linux/gentoo/ http://tux.rainside.sk/gentoo/ http://123.58.173.106/gentoo/ ftp://ftp.ntua.gr/pub/linux/gentoo/ http://mirrors.xservers.ro/gentoo/ http://123.58.173.89/gentoo/ ftp://ftp.lecl.net/pub/gentoo/ rsync://mirror.bytemark.co.uk/gentoo/ http://mirror.bytemark.co.uk/gentoo/ ftp://gentoo.lagis.at/"
