Disable password authentication for this server.

Index: ssh/sshd_config
===================================================================
--- ssh.orig/sshd_config
+++ ssh/sshd_config
@@ -65,7 +65,7 @@ Protocol 2
 
 # anchor_1i2w6k7g8dtsxb62ev3eliju4_5
 # To disable tunneled clear text passwords, change to no here!
-#PasswordAuthentication yes
+PasswordAuthentication no
 #PermitEmptyPasswords no
 
 # anchor_1i2w6k7g8dtsxb62ev3eliju4_6
