Defines a customized layout for the squid cache.

Index: squid/squid.conf
===================================================================
--- squid.orig/squid.conf
+++ squid/squid.conf
@@ -1921,6 +1921,7 @@ hierarchy_stoplist cgi-bin ?
 #
 #Default:
 # cache_dir ufs /var/cache/squid 100 16 256
+cache_dir ufs /var/cache/squid 500 32 1024
 
 #  TAG: store_dir_select_algorithm
 #	Set this to 'round-robin' as an alternative.
