Disables proxy access by neighbour caches.

Index: squid/squid.conf
===================================================================
--- squid.orig/squid.conf
+++ squid/squid.conf
@@ -694,7 +694,7 @@ http_access deny all
 # icp_access deny all
 #
 #Allow ICP queries from everyone
-icp_access allow all
+#icp_access allow all
 
 #  TAG: htcp_access
 #	Allowing or Denying access to the HTCP port based on defined
@@ -3553,6 +3553,7 @@ broken_vary_encoding allow apache
 #
 #Default:
 # icp_port 3130
+icp_port 0
 
 #  TAG: htcp_port
 #	The port number where Squid sends and receives HTCP queries to
