Disables proxy access by neighbour caches.

Index: squid/squid.conf
===================================================================
--- squid.orig/squid.conf
+++ squid/squid.conf
@@ -728,7 +728,7 @@ http_access deny all
 # icp_access deny all
 #
 #Allow ICP queries from local networks only
-icp_access allow localnet
+#icp_access allow localnet
 icp_access deny all
 
 #  TAG: htcp_access
@@ -3905,6 +3905,7 @@ broken_vary_encoding allow apache
 #
 #Default:
 # icp_port 3130
+icp_port 0
 
 #  TAG: htcp_port
 #	The port number where Squid sends and receives HTCP queries to
