Disables proxy access by neighbour caches.

Index: squid/squid.conf
===================================================================
--- squid.orig/squid.conf
+++ squid/squid.conf
@@ -698,7 +698,7 @@ http_access deny all
 # icp_access deny all
 #
 #Allow ICP queries from local networks only
-icp_access allow localnet
+#icp_access allow localnet
 icp_access deny all
 
 #  TAG: htcp_access
@@ -3594,7 +3594,7 @@ refresh_pattern .		0	20%	4320
 #Default:
 # icp_port 0
 #
-icp_port 3130
+#icp_port 3130
 
 #  TAG: htcp_port
 #	The port number where Squid sends and receives HTCP queries to
