Defines ACL list for realm-specfic SSH ports.

Index: squid/squid.conf
===================================================================
--- squid.orig/squid.conf
+++ squid/squid.conf
@@ -5875,6 +5875,7 @@ acl Safe_ports port 591		# filemaker
 acl Safe_ports port 777		# multiling http
 acl Safe_ports port 901		# SWAT
 acl CONNECT method CONNECT
+acl ssh_ports port 22 874-877 992
 
 http_access allow manager from_localhost
 http_access deny manager
